lib/lightning/projects/policy.ex

```diff
@@ -8,6 +8,12 @@ defmodule Lightning.Projects.Policy do
   alias Lightning.Projects
   alias Lightning.Projects.Project
 
+  # Users with admin level access to a project ccan access project settings with read/write rights
+  def authorize(action, %User{} = user, %Project{} = project)
+      when action in [:access_project_settings] do
```

Comment:
No need for a guard clause here, can do this:

```
def authorize(:access_project_settings, %User{} = user, %Project{} = project)
```

We only use guard clauses (in these places) when more than one action has the same authorize logic, like a case statement but for function matches.

Also why is it called `access_project_settings` when a normal user can still see the settings - should it not be `change_project_settings` or something like that?

Reply:
I turned it to `:edit` and removed the guard clause. See my last commit. Thanks